libs/coin-bitcoin/src/api/index.ts

```diff
@@ -55,10 +55,10 @@ export const getFeeItems = async (currency: CryptoCurrency): Promise<FeeItems> =
     const blockCount = parseInt(key, 10);
     if (blockCount === defaultBlockCount) defaultFeePerByte = feePerByte;
 
-    if (!Number.isNaN(blockCount) && !feePerByte.isNaN() && blockCount in speeds) {
+    if (!Number.isNaN(blockCount) && !feePerByte.isNaN() && speeds.has(blockCount)) {
```

Comment:
not directly related to the PR but it is a tech debt
this getFeeItems is not correct.
fees item can be an integer other than 1,3,6 (e.g. https://explorers.api.live.ledger.com/blockchain/v4/kmd/fees)
the "getFeeItems" is now only used in mock and duplicated with the logic in "bitcoin/getAccountNetworkInfo.ts"
we can removed completely this method and use same logic of "js-prepareTransaction.ts"

Reply:
Sorry, but it breaks some test logic ("libcore" account Need Sync error). I prefer not refacto the test and rely on them for this PR.